databricks/koalas/frame.py

```diff
@@ -2626,9 +2671,22 @@ def describe(self) -> 'DataFrame':
         if len(exprs) == 0:
             raise ValueError("Cannot describe a DataFrame without columns")
 
-        sdf = self._sdf.select(*exprs).describe()
-        return DataFrame(sdf, index=Metadata(data_columns=data_columns,
-                                             index_map=[('summary', None)])).astype('float64')
+        if percentiles is not None:
+            if any((p < 0.0) or (p > 1.0) for p in percentiles):
+                raise ValueError("Percentiles should all be in the interval [0, 1]")
+            # appending 50% if not in percentiles already
+            percentiles = (percentiles + [0.5]) if 0.5 not in percentiles else percentiles
+        else:
+            percentiles = [0.25, 0.5, 0.75]
+
+        formatted_perc = ["{:.0%}".format(p) for p in sorted(percentiles)]
+        stats = ["count", "mean", "stddev", "min", *formatted_perc, "max"]
+
+        sdf = self._sdf.select(*exprs).summary(stats)
+
+        return DataFrame(sdf.replace("stddev", "std", subset='summary'),
+                         index=Metadata(data_columns=data_columns,
+                         index_map=[('summary', None)])).astype('float64')
```

Comment:
I'm wondering why lint-python doesn't raise an error, but `index_map` should be the same indent as `data_columns`?